optimum/intel/openvino/configuration.py

```diff
@@ -40,9 +40,10 @@ class OVQuantizationMethod(str, Enum):
 
 
 _DEFAULT_4BIT_CONFIGS = {
-    "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "scale_estimation": True},
+    "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "ratio": 0.8},
     "EleutherAI/gpt-j-6b": {"bits": 4, "sym": False, "group_size": 64},
     "facebook/opt-6.7b": {"bits": 4, "sym": False, "group_size": 64, "ratio": 0.8},
+    "bigscience/bloomz-7b1": {"bits": 4, "sym": False, "group_size": 32, "ratio": 0.6},
```

Comment:
Missed revert.

Reply:
actually removing it in https://github.com/huggingface/optimum-intel/pull/824/ broke this test https://github.com/huggingface/optimum-intel/blob/31430d6e1613ec8cf42a00151a7f687aa9933a8c/tests/openvino/test_exporters_cli.py#L254

Reply:
Would you like me to do something with this? This configuration is outdated for the original model. As I can see, it is not possible to use any other model since configuration can not be applicable due to group size.

Reply:
Could it be patched as done in https://github.com/huggingface/optimum-intel/blob/31430d6e1613ec8cf42a00151a7f687aa9933a8c/tests/openvino/test_quantization.py#L443